modules/oadp-about-backing-and-restoring-from-cluster-to-cluster.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+
+:_content-type: CONCEPT
+[id="oadp-about-backing-and-restoring-from-cluster-to-cluster_{context}"]
+= About backing up data from one cluster and restoring it on another cluster
+
+OpenShift API for Data Protection (OADP) is designed to back up and restore application data in the same {product-title} cluster. {mtc-full} ({mtc-short}) is designed to migrate data from one {product-title} cluster to another or to migrate data from a VMware cluster to an {product-title} cluster.
+
+You can use OADP to back application data from one cluster and restore it on another. However, doing so is both more complicated than using {mtc-short} to do so or using OADP to back up and restore to the same {product-title} cluster.
+
+To successfully use OADP to back up data from one cluster and restore it to another, you must take into account the following factors, in addition to the prerequisites and procedures that apply to using OADP to back up and restore data on the same cluster:
+
+* Operators
+* Use of Velero
+* UID and GID ranges
+
+[id="oadp-cluster-to-cluster-operators_{context}"]
+== Operators
+You must exclude Operators from the backup of an application for backup and restore to succeed.
+
+[id="oadp-cluster-to-cluster-velero_{context}"]
+== Use of Velero
+
+* Velero, which OADP is built upon, does not natively support migrating persistent volumes snapshots across cloud providers. To migrate volume snapshot data between cloud platforms, you must enable Velero's link:https://velero.io/docs/main/file-system-backup/[File System Backup] option which backs up volume contents at the filesystem level, enabling the desired migration.
```

Comment:
```suggestion
* Velero, which OADP is built upon, does not natively support migrating persistent volumes snapshots across cloud providers. To migrate volume snapshot data between cloud platforms, you must enable restic option (known as file-system-backup in Velero) which backs up volume contents at the filesystem level, or use data mover functionality enabling the desired volume migration.
```

references:
- [Enabling Data Mover in OADP Operator](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html-single/backup_and_restore/index#oadp-using-data-mover-for-csi-snapshots_backing-up-applications)
- [Enabling restic in OADP Operator](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html-single/backup_and_restore/index#oadp-backing-up-applications-restic_backing-up-applications) also known as file system backup as of OADP 1.2.0

Reply:
velero CLI example
https://velero.io/docs/main/file-system-backup/#how-velero-integrates-with-restic:~:text=velero%20backup%20create%20BACKUP_NAME%20%2D%2Ddefault%2Dvolumes%2Dto%2Dfs%2Dbackup%20OTHER_OPTIONS

Reply:
@kaovilai  I added the references you suggested. Are you asking to also add step 2  of https://velero.io/docs/main/file-system-backup/#using-the-opt-out-approach  or the whole section? FYI, I'll need to get QE to verify whatever procedures I add.